src/gbytec.f

```diff
@@ -1,4 +1,14 @@
 C> @file
+C> @brief Wrapper for gbytesc() limiting NSKIP and N to 0 and 1
```

Comment:
Punctuation...

Reply:
NSKIP and N are variables in gbytesc() should variable names all be lowercase?

Reply:
No it should have a period at end of sentence. ;-)